### DISCUSSION TITLE
播放器唱片模式下的歌曲名称歌手信息是否可以和封面模式一样放到左侧的下面？

### DISCUSSION THREAD

**Discussion by jcfun** [SPlayer-Dev/SPlayer#290] [想法 Ideas]
![Snipaste_2024-11-07_11-42-32](https://github.com/user-attachments/assets/9a08b746-f3b1-4a39-8b8e-5d96d2f8ad53)
如上图，封面模式下歌曲名称歌手信息在左侧下方，右侧的歌词可以显示更多的内容，同时歌词的水平居中显示是在居于整个页面水平居中的，这样比较协调。

![Snipaste_2024-11-07_11-41-21](https://github.com/user-attachments/assets/d33da996-4109-4410-b5a2-b77a8de8cb65)
在唱片模式下，歌曲名称歌手信息被移动到了歌词上方，这样歌词的显示区域就变小了，特别是在歌词同时有原词和翻译的情况下，可能只能显示3行。而且歌词的水平居中显示变为了歌词显示区域的水平居中，观感上并不是那么协调。

**Comment by imsyy** [comment]
曾经是在左边的，但是感觉不太好看就移到右边了，看来目前的方案也不太行
主要是我平时也没用过歌词居中滚动和居中，没发现不协调 😂

**Comment by imsyy** [comment]
![image](https://github.com/user-attachments/assets/61c35f73-2759-439a-ae4b-3930b04ffd63)

感觉可以像评论模式时的这样

**Reply by jcfun** [reply]
是的，这样比较合适

**Reply by imsyy** [reply]
> 是的，这样比较合适

已调整
